include/lmms_math.h

```diff
@@ -42,20 +42,20 @@ static inline bool approximatelyEqual(float x, float y)
 	return x == y ? true : std::abs(x - y) < F_EPSILON;
 }
 
-#ifdef __INTEL_COMPILER
-
-static inline float absFraction( const float _x )
-{
-	return( _x - floorf( _x ) );
-}
-
-static inline float fraction( const float _x )
+/*!
+ * @brief Returns the fractional part of a float, a value between -1.0f and 1.0f.
+ *
+ * fraction( 2.3) =>  0.3
+ * fraction(-2.3) => -0.3
+ *
+ * Note that if the return value is used as a phase of an oscillator, that the oscillator must support
+ * negative phases.
+ */
+static inline float fraction(const float x)
```

Comment:
A lot of functions in this header are marked both `static` and `inline` which is pretty strange. They should all probably just be `inline`.

Reply:
So get rid of static?

Reply:
And the functions that can be marked `constexpr` should probably use that. For those functions, `inline` won't be needed since `constexpr` implies `inline` for functions.

Reply:
Yes, constexpr functions are just like constexpr constants in that they can be used in both compile-time and runtime contexts. Unfortunately most of `<cmath>` is not constexpr until C++23, so functions in this header that use cmath can't be marked constexpr yet. So I guess just make functions `inline` (without the `static`) for now.